src/iOS/readme.md

```diff
@@ -2,258 +2,258 @@
 
 ## Analysis
 
-* [Distance measurement analysis](Xamarin.iOS/Samples/Analysis/DistanceMeasurement/readme.md) - Measure distances between two points in 3D.
-* [Line of sight (geoelement)](Xamarin.iOS/Samples/Analysis/LineOfSightGeoElement/readme.md) - Show a line of sight between two moving objects.
-* [Line of sight (location)](Xamarin.iOS/Samples/Analysis/LineOfSightLocation/readme.md) - Perform a line of sight analysis between two points in real time.
-* [Query feature count and extent](Xamarin.iOS/Samples/Analysis/QueryFeatureCountAndExtent/readme.md) - Zoom to features matching a query and count the features in the current visible extent.
-* [Viewshed for camera](Xamarin.iOS/Samples/Analysis/ViewshedCamera/readme.md) - Analyze the viewshed for a camera. A viewshed shows the visible and obstructed areas from an observer's vantage point. 
-* [Viewshed for GeoElement](Xamarin.iOS/Samples/Analysis/ViewshedGeoElement/readme.md) - Analyze the viewshed for an object (GeoElement) in a scene.
-* [Viewshed (location)](Xamarin.iOS/Samples/Analysis/ViewshedLocation/readme.md) - Perform a viewshed analysis from a defined vantage point.
+* [Distance measurement analysis](Xamarin.iOS/Samples/Analysis/DistanceMeasurement) - Measure distances between two points in 3D.
```

Comment:
Not sure if it matters, but since the link is to a folder, should the final `/` be included in the URL?
>`[Distance measurement analysis](Xamarin.iOS/Samples/Analysis/DistanceMeasurement/)`